<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfficeDialog</class>
 <widget class="QDialog" name="OfficeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый офис</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QCheckBox" name="enabledCheckBox">
       <property name="text">
        <string>Разрешён</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="ctimeLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="performerBillPercentLabel">
       <property name="text">
        <string>Процент от суммы мастера</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="performerBillPercentSpinBox">
       <property name="toolTip">
        <string>Процент от суммы, на который выставляется счёт</string>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="urlLabel">
       <property name="text">
        <string>Сайт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="urlLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="phone1Label">
       <property name="text">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phone1LineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="phone2Label">
       <property name="text">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phone2LineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="addressLabel">
     <property name="text">
      <string>Адрес</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="addressPlainTextEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="managernameLabel">
       <property name="text">
        <string>Ф.И.О. менеджера</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="managernameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="notesPlainTextEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="icon">
        <iconset resource="kcrm.qrc">
         <normaloff>:/images/ok32.png</normaloff>:/images/ok32.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="kcrm.qrc"/>
 </resources>
 <connections/>
</ui>
